torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -567,6 +567,19 @@ RegisterOperators reg(
            return 0;
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "prim::grad(Tensor a) -> Tensor(*)",
+         [](Stack& stack) {
+           at::Tensor a;
+           pop(stack, a);
+           push(stack, a.grad());
+           return 0;
+         },
+         aliasAnalysisFromSchema()),
+     Operator(
+         "prim::data(Tensor(b) a) -> Tensor(b)",
+         noop,
```

Comment:
Why is this a noop? 
```
> a = torch.tensor(.5)
> a is a.data
False
```

Reply:
this is because python binds are different from the underlying C++, https://github.com/pytorch/pytorch/blob/c5482e33e9ef59af6c18059378964a00036d95aa/torch/csrc/autograd/python_variable.cpp#L235 python side re-wrap the underlying `at::Tensor` again in a new THPVariable, so the identity check in python is different, but the underlying C++ `at::Tensor` (also in our interpreter) is still the same.